libraries/apollo-runtime/src/jvmCommonMain/kotlin/com/apollographql/apollo/network/OkHttpExtensions.kt

```diff
@@ -16,6 +19,18 @@ import okhttp3.OkHttpClient
  *
  * See also [ApolloClient.Builder.httpEngine] and [ApolloClient.Builder.webSocketEngine]
  */
+@Deprecated(
+    "Use networkTransport() instead",
+    ReplaceWith(
+        "networkTransport(HttpNetworkTransport.Builder().httpEngine(DefaultHttpEngine(okHttpClient)).build())" +
+            ".subscriptionNetworkTransport(WebSocketNetworkTransport.Builder().webSocketEngine(DefaultWebSocketEngine(okHttpClient)).build())",
+        "com.apollographql.apollo.network.http.HttpNetworkTransport",
+        "com.apollographql.apollo.network.http.DefaultHttpEngine",
+        "com.apollographql.apollo.network.ws.DefaultWebSocketEngine",
+        "com.apollographql.apollo.network.ws.WebSocketNetworkTransport"
+    )
+)
+@ApolloDeprecatedSince(ApolloDeprecatedSince.Version.v4_1_1)
```

Comment:
I worry a bit about this one in particular as it's probably one the most often used, and the replacement expression is a bit of mouthful... But I agree about the tradeoff, it's for the better.

Reply:
That's a fair point. We don't want to become the `capitalized()` of GraphQL... Will think more about this